backup-configs/airflow-backup-configs.py

```diff
@@ -24,7 +24,7 @@
 ALERT_EMAIL_ADDRESSES = []
 # Format options: https://www.tutorialspoint.com/python/time_strftime.htm
 BACKUP_FOLDER_DATE_FORMAT = "%Y%m%d%H%M%S"
-BACKUP_HOME_DIRECTORY = "/tmp/airflow_backups"
+BACKUP_HOME_DIRECTORY = Variable.get("airflow_maintenance_dags_backup_config__directory", "/tmp/airflow_backups")
```

Comment:
Please change the Variable name from `airflow_maintenance_dags_backup_config__directory` to `airflow_backup_config__backup_home_directory` for consistency.

Reply:
Done, thanks @prakshalj0512